plugins/channelrx/radioclock/radioclocksink.cpp

```diff
@@ -28,7 +28,6 @@
 
 RadioClockSink::RadioClockSink(RadioClock *radioClock) :
         m_scopeSink(nullptr),
-        m_radioClock(radioClock),
```

Comment:
Remove from constructor parameters if not needed

Reply:
ok